Bottom: 56ef8862a6a93a9854916b06f89873c9e093ae89
Top:    6759887c37d7ca97e9ce0a6ebafdf65fe30ec428
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:13 +0000

Bump circle approximation detail level for gerber export


---

diff --git a/src/polygon.h b/src/polygon.h
index 58634f7..90d0d26 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 24 //20 //8 //40
+#define POLY_CIRC_SEGS 40 //24 //20 //8 //40
 #define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
